db/system_keyspace.cc

```diff
@@ -2740,10 +2743,10 @@ class cluster_describe_table : public streaming_virtual_table {
 
             co_await result.emit_partition_start(make_partition_key(ks_name));
             clustering_row cr(clustering_key::make_empty());
-            set_cell(cr.cells(), to_bytes("cluster"), cluster);
-            set_cell(cr.cells(), to_bytes("partitioner"), partitioner);
-            set_cell(cr.cells(), to_bytes("snitch"), snitch);
-            set_cell(cr.cells(), to_bytes("range_ownership"), map_value);
+            set_cell(cr.cells(), to_bytes(desc::CLUSTER_COLUMN), cluster);
+            set_cell(cr.cells(), to_bytes(desc::PARTITIONER_COLUMN), partitioner);
+            set_cell(cr.cells(), to_bytes(desc::SNITCH_COLUMN), snitch);
+            set_cell(cr.cells(), to_bytes(desc::RANGE_OWNERSHIP_COLUMN), map_value);
```

Comment:
A better way to break this cycle would be:
* add skeleton `cql3::statements::describe_statement` with the constants
* implement virtual tables using said constants
* finish implementing statement classes

Reply:
Cycle broken, it was a pain indeed